controllers/utils/utils.go

```diff
@@ -162,7 +161,25 @@ func VerifyIfCapabilityIsEnabled(ctx context.Context, cli client.Client, capabil
 	}
 
 	return false, nil
+}
 
+// GetDSCIName returns the name of the DSCI in the cluster. If it fails to read the DSCI,
+// it returns the default DSCI name.
+func GetDSCIName(ctx context.Context, cli client.Client) (string, error) {
```

Comment:
`DSCIName` would imply the name of the DSCI resource right? Whereas what we are looking for is the name of the control plane. WDYT about `GetIstioControlPlaneName` instead?

Reply:
Yeah, that sounds good.